test/caesar_test.rb

```diff
@@ -1,2 +1,39 @@
 # YOUR TESTS GOES HERE
-# I PITY THE FOOL WHO DOESNT WRITE TESTS
+# # I PITY THE FOOL WHO DOESNT WRITE TESTS
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/caesar'
+
+class CaesarTest < MiniTest::Test
+
+def setup
+  @caesar = Caesar.new
+end
+
+def test_it_can_cipher
+  coded = @caesar.eng_to_cipher("the quick brown fox jumps over the lazy dog", 3)
+  assert_equal "qeb nrfzh yoltk clu grjmp lsbo qeb ixwv ald", coded
+end
+
+# Iteration 1
+def test_it_can_handle_cases
+  coded = @caesar.eng_to_cipher("The quick brown fox JUMPS over the lazy DOG", 3)
+  assert_equal "Qeb nrfzh yoltk clu GRJMP lsbo qeb ixwv ALD", coded
+end
+
+# Iteration 2
+# Translate English to ciphertext from a file
+def test_it_can_cipher_a_file
+  file = File.open("input.txt", "w")
+  file.write("I am in a file")
```

Comment:
Interesting choice to write to the file. This ensures that this test will work even if the file isn't there. Great job!